pkgs/development/cuda-modules/setup-hooks/extension.nix

```diff
@@ -32,18 +32,6 @@ final: _: {
       {}
     );
 
-  autoAddOpenGLRunpathHook =
```

Comment:
Looking at
https://gist.github.com/GrahamcOfBorg/5e54dc934ffe416fcb2645c003afdc4c#file-output-path-comparison-L60 (and also to stay compatible for a while) we'll need to

`autoAddOpenGLRunpathHook = final.pkgs.autoAddDriverRunpathHook` in `cudaPackages`

Reply:
That's odd. I already do this here: [`651a917` (#275241)](https://github.com/NixOS/nixpkgs/pull/275241/commits/651a917b9015ca2cb5e6870cc6f7c0b649989be3#diff-b4db74011c11cf9da1bf5ea678b9f161d1c930357ee2c04f2cce211fcaea60f4R52-R55)